### DISCUSSION TITLE
What causes horrible results like this?

### DISCUSSION THREAD

**Discussion by githubUser01946** [zero01101/openOutpaint#191] [Q&A]
First off the prompt is there and the correct inpainting model is selected.

![image](https://user-images.githubusercontent.com/117453319/215812078-a646644b-5c69-4b50-8bed-c0d3864d0db0.png)

**Comment by githubUser01946** [comment]
Also, can I run this outside of Webui?

**Comment by zero01101** [comment]
generally you'd see results like that when the requested resolution is _way_ too small for the output size/reticle, i.e. you've got sync resolution disabled and a really low resolution setting (or if using the dream tool, you've left HRfix enabled over a completely empty region but this looks like you've already got image content so it's probably not txt2img HRfix related)

>Also, can I run this outside of Webui?

openOutpaint actually started as a standalone webpage; the extension is merely a convenience :) just download the zip of the codebase

![image](https://user-images.githubusercontent.com/1649724/215901329-ce873917-8876-45dc-a7e2-6a652295a2ad.png)

extract it to a new folder somewhere, [adjust your webUI user script accordingly](https://github.com/zero01101/openOutpaint/wiki/Manual#requirements), run webUI, run the included openOutpaint launch script, and navigate to http://127.0.0.1:3456

**Reply by githubUser01946** [reply]
Ok so how do I fix this?

**Reply by zero01101** [reply]
my recommendations would be ensuring you're using synced resolution/reticle size (in general really but mentioned for img2img because of your posted image), or making sure you've got reasonable settings for HRfix and disabling it if you're not using it in txt2img; aside from that, uh... maybe make sure you're using the right VAE? 

regarding the resolution, you can check the parameters sent to SD in your browser's console as well as the raw POST request in the networking tab to check that the requested params are what you've input, but yeah, really we don't have much control over what SD gives us back in response to the request parameters

**Comment by githubUser01946** [comment]
There's only one vae installed, the usual one most people have.

**Reply by zero01101** [reply]
i couldn't even begin to guess what the "usual one most people have" would be given the incredible number of models floating around out there, some in .ckpt, some in .safetensors, some with VAEs baked in, some not, but regardless, without further information i'm not sure what to tell you; like i said, ensure you're using appropriate resolutions for your requests to the API either by the sync res checkbox or HRfix... also as mentioned, you can check the parameters received as response from the API from your browser's devtools console tab 

![image](https://user-images.githubusercontent.com/1649724/216466549-2cc4d5d7-83c4-4c0d-88d0-e810c4bea58f.png)

or the params you sent via the network tab; look for the POST request to `txt2img` or `img2img` 

![image](https://user-images.githubusercontent.com/1649724/216466629-cdf0bc68-b163-42b3-9a4d-6ad9fe096129.png)

**Reply by zero01101** [reply]
yeah i'm really going to guess it's something like this 
![image](https://user-images.githubusercontent.com/1649724/216472006-56e0174c-7566-4691-a77b-cdb23366c573.png)

**Comment by ghost** [comment]
I had these responses when I was using "highres fix". Deactivate it, and it will render normal images on no background at all. If you've done it, select the sd-1.5 inpainting model.

**Comment by githubUser01946** [comment]
I didn't have that ticked but it does seem to have gone away.

At least I know now what to do if it happens again.

**Comment by joshyy1312** [comment]
I got this when I dream/img2img a zone smaller than 384px in any direction
